### DISCUSSION TITLE
When would WET upgrade to Bootstrap version 5.0

### DISCUSSION THREAD

**Discussion by rogercote8** [wet-boew/wet-boew#9528] [Q&A]
We are users of Microsoft Power Pages (Portal).  Currently Power Pages is running under Bootstrap version 3.3.  The latest version of WET is running under version 4.4.  We have issues with Power Pages when we try to use the CDTS running version, as the Bootstrap version are not the same .  We have to use CDTS v4.0.43 to match with Power Pages.  

Power Pages is planning to upgrade to Bootstrap version 5.0 in 2023.  

**Question:**  When will WET upgrade to Bootstrap version 5.0.  

We are concerned that if Power Pages upgrade to Bootstrap version 5.0 ahead of WET, we might run into unforeseen issues.

**Comment by ghost** [comment]
See https://github.com/wet-boew/wet-boew/issues/9508#issuecomment-1416427855

###### I am just a volunteer on this GitHub project.
